settings.yaml

```diff
@@ -0,0 +1,123 @@
+HighFrequencyTradingService:
+    Db:
+        LogsConnString:
+            settings-key: LogsConnString
+            types:
+                - AzureBlobStorage
+                - Tag
+        OrderStateConnString:
+            settings-key: HFT-OrderStateConnString
+            types:
+                - MongoDB
+        OrdersArchiveConnString:
+            settings-key: HFT-OrdersArchiveConnString
+            types:
+                - AzureTableStorage
+    
+    MaintenanceMode:
+        Enabled: 
+            settings-key: HFT-MaintenanceMode
+        Reason:
+            settings-key: HFT-MaintenanceReason
+
+    DisabledAssets:
+        settings-key: HFT-DisabledAssets
+        types:
+            - JsonArray
+
+    Dictionaries:
+        CacheExpirationPeriod:
+            settings-key: HFT-DictionaryCacheExpirationPeriod
+
+    Cache:
+        RedisConfiguration:
+            settings-key: HFT-RedisConfiguration
+            types:
+                - Redis
+        ApiKeyCacheInstance:
```

Comment:
I would rename it to something more meaningful, like "ApiKeysPrefix". Or even hardcoded it. I don't see right now why devops should have an ability to change this prefix.